### PR TITLE
12.0 1282 cleanup

### PR DESCRIPTION
**Pull request by trojikman**
from #1282

### CODE REVIEW

File: pos_invoice_pay/models.py [hunk 1]
```diff
@@ -177,6 +251,11 @@ def _get_default_writeoff_account(self):
         help="Ask for a cashier when fetch orders",
         defaul=True,
     )
+    invoice_pos_order = fields.Boolean(
+        string="Create POS Order after Invoices",
+        help="When you pay an invoice via POS a pos order, based on invoice date will be created",
```
Comment (yelizariev) [inline pos_invoice_pay/models.py]: Предложение не читается

```suggestion
        help="When you pay an invoice via POS a pos order will be created",
```

**Review by yelizariev — COMMENTED** [review]
Добавь doc/index.rst
Так не понятно для чего это

**Comment by trojikman** [comment]
I approve to merge it now

**Comment by trojikman** [comment]
via Hubstaff
User: Denis Mudarisov

Project: pos-addons - https://app.hubstaff.com/projects/1618281
Date Range: 09/07/21 - 09/07/21
Work session total: 0:17:03
Billable: Yes

Grand total: 0:17:03

**Comment by trojikman** [comment]
via Hubstaff
User: Denis Mudarisov

Project: pos-addons - https://app.hubstaff.com/projects/1618281
Date Range: 09/07/21 - 09/07/21
Work session total: 0:02:59
Billable: Yes

Grand total: 0:20:02

**Pull request merged**